### PR TITLE
Add io.github.riftr.intelpy

### PR DESCRIPTION
**Pull request by Riftr**
### Please confirm your submission meets all the criteria

- [x] Please describe your application briefly.
 IntelPy is a cross-platform chat log monitor and alert system for the game Eve Online.

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I have [built][build] and tested the submission locally.
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an author/developer/upstream contributor of the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [x] The domain used for the application ID is controlled by the application developers either directly or through the code hosting (e.g. GitHub, GitLab, SourceForge, etc.). The [application id guidelines][app-id] are followed.
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://docs.flathub.org/docs/for-app-authors/requirements
[maint]: https://docs.flathub.org/docs/for-app-authors/maintanance
[submission]: https://docs.flathub.org/docs/for-app-authors/submission
[build]: https://docs.flathub.org/docs/for-app-authors/submission/#before-submission
[app-id]: https://docs.flathub.org/docs/for-app-authors/requirements#application-id

### CODE REVIEW

File: io.github.riftr.intelpy.yml [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+command: run.sh
+finish-args:
+  - --share=ipc
+  - --device=dri
+  - --filesystem=~
+  - --socket=pulseaudio
+build-options:
+  env:
+    QT_DEBUG_PLUGINS: '1'
+
```
Comment (bbhtt) [inline io.github.riftr.intelpy.yml:17]: You need to cleanup the baseapp from development files https://github.com/flathub/com.riverbankcomputing.PyQt.BaseApp?tab=readme-ov-file#example-pyqtwebengine-application cleanup-commands.

**Comment by hfiguiere** [comment]
The top of the PR template say the PR should be made against the `new-pr` branch. This is not the case.

File: io.github.riftr.intelpy.yml [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+runtime-version: '5.15-23.08'
+sdk: org.kde.Sdk
+base: com.riverbankcomputing.PyQt.BaseApp
+base-version: 5.15-23.08
+name: intelpy
+command: run.sh
+finish-args:
+  - --share=ipc
+  - --device=dri
+  - --filesystem=~
```
Comment (hfiguiere) [inline io.github.riftr.intelpy.yml:12]: this is an invalid filesystem....

**Review by hfiguiere — CHANGES_REQUESTED** [review]
I don't see any of the required .desktop, icon and appstream file installed.

**Comment by Riftr** [comment]
> I don't see any of the required .desktop, icon and appstream file installed.

I've added these now. I had them ready in my own repo and I wasn't sure if I needed to specifically move them.

**Comment by bbhtt** [comment]
bot, build io.github.riftr.intelpy

**Comment by bbhtt** [comment]
YOu have to change the screenshots to raw.githubusercontent.com links like this https://raw.githubusercontent.com/Riftr/intelpy/master/intelpy/externals/Intelpy-2-main.png

It has to be a direct link, a link to a page won't work.

**Comment by Riftr** [comment]
> You have to change the screenshots to raw.githubusercontent.com links like this https://raw.githubusercontent.com/Riftr/intelpy/master/intelpy/externals/Intelpy-2-main.png
> 
> It has to be a direct link, a link to a page won't work.

I've made and pushed the change for this now too.

**Comment by bbhtt** [comment]
You have to update the tag/commit here.

**Comment by Riftr** [comment]
> You have to update the tag/commit here.

New release has been added here.

**Comment by bbhtt** [comment]
bot, build io.github.riftr.intelpy

**Comment by Riftr** [comment]
Accepted, thank you!

**Pull request closed without merging**